### ISSUE TITLE
Disabled users appear on user assignment

### ISSUE THREAD

**Issue by Gojo-Taqi** [avniproject/avni-webapp#1182]
**Describe the bug**
Disabled users appear on user assignment.

**To Reproduce**

1. Disable a user
2. Come to user assignment, and for any subject type, under the section assigned to, you must be able to see the disabled user.
3. Refer to the screenshot attached below.

![Image](https://github.com/avniproject/avni-webapp/assets/94247708/f61eb822-6cb2-4fc8-a203-3a1e065b8f33)

**Expected behavior**
No disabled users should be present there.

**Comment by mahalakshme** [comment]
@Gojo-Taqi thanks for creating these cards, please keep going, its helpful

**Comment by ALOK9442** [comment]
Hey @mahalakshme, I want to work on this issue, how to start the project locally.

**Comment by mahalakshme** [comment]
@ALOK9442 you can setup the below to get started:
https://avni.readme.io/docs/developer-environment-setup-ubuntu#postgresql
https://avni.readme.io/docs/developer-environment-setup-ubuntu#avni-server
https://avni.readme.io/docs/developer-environment-setup-ubuntu#java
https://avni.readme.io/docs/developer-environment-setup-ubuntu#web-app-components 

to start working

**Comment by ALOK9442** [comment]
sir I followed all these steps but  when I am starting the server i get error which you can see in the image I have attached also I am getting the same error with 'make start-with-staging' and 'yarn start-with-staging'
![Screenshot 2024-04-01 213012](https://github.com/avniproject/avni-webapp/assets/104527572/a98cbac8-579b-4ef3-b592-93859613ed72)

**Comment by ALOK9442** [comment]
> @ALOK9442 you should be using ```make start``` or ```make start-with-staging```. Look up the Makefile in the project

Yes sir, I am using the same command as you say but still getting that error.

**Comment by vinayvenu** [comment]
@ALOK9442 hmm. just noticed you are on Windows. All developers on the project are either on osx or linux, so it is possible this is the wrong way to set up environment variables on Windows. 

The basic idea here is to set up the environment variable ```BACKEND_URL``` and do a ```react-scripts --max_old_space_size=2048 --openssl-legacy-provider start```, which starts the node process for the app.

**Comment by ALOK9442** [comment]
Hello sir, could please tell me more about this react-scripts --max_old_space_size=2048 --openssl-legacy-provider start and also how to setup the env file

**Comment by ALOK9442** [comment]
Hello @vinayvenu sir, could please tell me more about this react-scripts --max_old_space_size=2048 --openssl-legacy-provider start and also how to setup the env file

**Comment by vinayvenu** [comment]
@ALOK9442 look up https://www.npmjs.com/package/react-scripts . It is core to React.

**Comment by ALOK9442** [comment]
Hello @vinayvenu sir, I still didn't understand what to do with the backend url and how to set it in .env file?

**Comment by ALOK9442** [comment]
> Hello @vinayvenu sir, I still didn't understand what to do with the backend url and how to set it in .env file?

hello  sir @vinayvenu , a gentle reminder

**Comment by ALOK9442** [comment]
Hello @vinayvenu sir,
I have did the setup and after running the server I am getting screen which totally confuses me as it says 
"There are multiple IDP types configured and you do not have an active session, please choose the one is used by your organisation"

could you help me with getting it started.
![Screenshot 2024-04-06 005930](https://github.com/avniproject/avni-webapp/assets/104527572/23d8506a-d209-498b-b178-811394fa8e3f)
![Screenshot (137)](https://github.com/avniproject/avni-webapp/assets/104527572/d19ec4e1-c5b6-466c-bcf4-ee91ea4cd893)
